Histogram.py

```diff
@@ -0,0 +1,169 @@
+import awkward as ak
+import pickle
+import numpy as np
+import hist
+import matplotlib as mpl
+from coffea.nanoevents import NanoEventsFactory, DelphesSchema
+
+def ET(vec):
+    return np.sqrt(vec.px**2+vec.py**2+vec.mass**2)
+
+def get_values(var,Events):
+    return ak.flatten(Events[var],axis=None)
+
+def fill_hist(var,nbins,bmin,bmax,label,Events):
+    h = (
+        hist.Hist.new
+        .Reg(nbins, bmin, bmax, label=label)
+        .Double()
+    )
+    h.fill(get_values(var,Events),weight=0.5)
+    return h
+
+def normalize_angle(angle):
+    angle = np.mod(angle, 2 * np.pi)
+    angle = np.where(angle >= np.pi, angle - 2 * np.pi, angle)
+    return angle
+
+def histogram(filename, model_name):                    
+    DelphesSchema.mixins["FatJet"] = "Jet"
+    Events = NanoEventsFactory.from_root(
+                file=filename,
+                    treepath="Delphes",
+                        schemaclass=DelphesSchema,
+                        ).events()
+    
+    # require two jets
+    events = Events
+    mask = ak.num(events.FatJet)>=2
+    events = events[mask]
+
+    # Dijet
+    events["Dijet"] = events.FatJet[:,0]+events.FatJet[:,1]
+
+    # transverse mass calculation
+    E1 = ET(events.Dijet)
+    E2 = events.MissingET.MET
+    MTsq = (E1+E2)**2-(events.Dijet.px+events.MissingET.px)**2-(events.Dijet.py+events.MissingET.py)
+    events["MT"] = np.sqrt(MTsq, where=MTsq>=0)
+            
+    # 4-vectors for dijet
+    events["Jet_pt"] = events.Dijet.pt
+    events["Jet_eta"] = events.Dijet.eta
+    events["Jet_phi"] = events.Dijet.phi
+    events["Jet_mass"] = events.Dijet.mass
+    # Other quantites to be plotted
+    events["MET"] = events.MissingET.MET
+    events["DeltaEta"] = events.FatJet.DeltaEta
+    events["DeltaPhi"] = events.FatJet.DeltaPhi
+
+    ## For plotting individually for jet1 and jet2
+    leading_jets = events.FatJet[:, :2]
+
+    jet1_mask = leading_jets[:, 0].pt >= leading_jets[:, 1].pt
+    jet2_mask = ~jet1_mask
```

Comment:
Jets should always be sorted by pt already, so you can just assign index 0 as Jet1 and index 1 as Jet2